src/frontend/parser.cpp

```diff
@@ -1454,14 +1491,18 @@ namespace graphit {
 // call_expr: ident ['<' endpoints '>'] '(' [expr_params] ')'
```

Comment:
Update the comment here to reflect the new syntax.

Reply:
yep thanks :)

Reply:
yep thanks